### PR TITLE
scroll-navbar-added

### LINKED ISSUE #2746: Scroll Navbar Project Under Navbar Section
**Describe the project you want to add with tech stack**
I would like to create a navbar which get's sticky and changes it's background color when we scroll down. I would like to add this 
project under Navbar section.

**Expected behavior**
This navbar will get sticky and will change it's background color as soon we scroll down to it's height

**Screenshots (optional)**

https://github.com/pranjay-poddar/Dev-Geeks/assets/123585591/7511c967-40c6-4f61-8903-3b5e04a71346

**Additional context (optional )**
I would like to work on this issue , @MohitGupta121  can you please assign it to me.

### PR DESCRIPTION
**Pull request by rahulrana701**
Scroll Navbar Project Under Navbar Section #2746

Close #2746

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by rahulrana701** [comment]
hi @MohitGupta121 added the scroll-navbar project

File: Js-Projects/Giveaway
Comment (MohitGupta121) [inline Js-Projects/Giveaway:1]: Why you adding this giveaway project in your PR. It's not part of this PR. Please remove this from here

File: Navbars/Scroll/tree-832079.jpg
Comment (MohitGupta121) [inline Navbars/Scroll/tree-832079.jpg:1]: Use image from Link not directly in your PR. Upload image somewhere and use it's link

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by rahulrana701** [comment]
dont' know why it is happening I had pushed only scroll navbar project not this giveaway project .
But I will create a new pr for this as well

**Comment by MohitGupta121** [comment]
Then I closing this PR create a new clean PR.

**Pull request closed without merging**